### DISCUSSION TITLE
I have a problem when using Auto-py-to-exe, this is a recurring error. Customtkinter and CTkToplevel.

### DISCUSSION THREAD

**Discussion by CeContreras** [TomSchimansky/CustomTkinter#1701] [Q&A]
![image](https://github.com/TomSchimansky/CustomTkinter/assets/42829956/1663d3fb-a267-4382-9b3a-2c7203fbd3d3)
when I compile the code it appears this message.

**Comment by Akascape** [comment]
@CeContreras Check this: #939

**Reply by CeContreras** [reply]
I see the solution but my error continues, I configure all the settings but not. You can help me?

**Reply by CeContreras** [reply]
![image](https://github.com/TomSchimansky/CustomTkinter/assets/42829956/22a90dc3-e552-4310-b074-4fc79e8cd04f)

**Reply by Akascape** [reply]
@CeContreras 
Did you properly pack the customtkinter folder and Ctkmessagebox folder in pyinstaller?

**Reply by CeContreras** [reply]
![image](https://github.com/TomSchimansky/CustomTkinter/assets/42829956/dbb275f6-9b46-4841-95de-49e0ac1ed3f9)
these are parameters that i do for create setup.
![image](https://github.com/TomSchimansky/CustomTkinter/assets/42829956/14da3a00-66fc-415b-a189-bc03fa7fd58f)
this is result

**Reply by Akascape** [reply]
@CeContreras Paste the customtkinter and Ctkmessagebox folder in the same directory where your program is also present. Then try.
